_drafts/contribute-to-ansible.markdown

```diff
@@ -38,10 +42,23 @@ For your information, this is a very interesting part to contribute on Ansible.
 
 ## Why contribute?
```

Comment:
as commiters, they are members of the core team

Reply:
done!